tests/periph_uart/main.c

```diff
@@ -57,16 +57,97 @@ static int parse_dev(char *arg)
 {
     unsigned dev = atoi(arg);
     if (dev >= UART_NUMOF) {
-        printf("Error: Invalid UART_DEV device specified (%u).\n", dev);
+        printf("msg: Invalid UART_DEV device specified (%u).\n", dev);
```

Comment:
unrelated? And inconsistent with the rest of the file?

Reply:
It is just so it can adhere to the riot_pal interface and RDM.